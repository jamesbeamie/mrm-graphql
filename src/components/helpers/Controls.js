import React from 'react';
import PropTypes from 'prop-types';
import '../../assets/styles/input.scss';
import { chevronIcon } from '../../utils/images/images';
/**
 *
 * The controls component
 *
 * @param {Object} controlsObject
 *
 * @returns {JSX}
 */
const Controls = ({ handleIncrement, controlsClass }) => (
  <div className={`controls ${controlsClass}`}>
    <button className="control up" onClick={handleIncrement} name="up">
      <img src={chevronIcon} alt="up" name="up" />
    </button>
    <button className="control down" onClick={handleIncrement} name="down">
      <img src={chevronIcon} alt="down" name="down" />
    </button>
  </div>
);
Controls.propTypes = {
  handleIncrement: PropTypes.func.isRequired,
  controlsClass: PropTypes.string,
};

Controls.defaultProps = {
  controlsClass: '',
};
export default Controls;
