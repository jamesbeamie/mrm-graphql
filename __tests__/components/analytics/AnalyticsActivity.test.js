import React from 'react';
import { mount } from 'enzyme';
import moment from 'moment';
import AnalyticsActivityComponent from '../../../src/containers/AnalyticsActivity';
import * as QueryHelper from '../../../src/components/helpers/QueriesHelpers';

moment.suppressDeprecationWarnings = true;
jest.mock('../../../src/components/helpers/QueriesHelpers');

describe('AnalyticsActivity component', () => {
  let wrapper;
  const props = {
    queryCompleted: jest.fn(),
    dateValue: {
      startDate: moment('2019 29 Apr', 'YYYY-MM-DD'),
      endDate: moment('2019 29 Apr', 'YYYY-MM-DD'),
    },
  };

  const userData = {
    id: '41',
    location: 'Nairobi',
    firstName: 'James',
  };

  const data = {
    allEvents: {
      events: [
        {
          room: {
            name: 'Test Room',
          },
          eventTitle: 'Ebot / Chrispine Feedback Sync',
          endTime: '2019-06-14T04:00:00-07:00',
          checkedIn: false,
          startTime: '2019-06-14T03:30:00-07:00',
          checkInTime: null,
          cancelled: false,
          numberOfParticipants: 3,
        },
        {
          room: {
            name: 'Test Room',
          },
          eventTitle: 'Robert <> Chrispine Feedback Sync',
          endTime: '2019-06-14T02:30:00-07:00',
          checkedIn: false,
          startTime: '2019-06-14T02:00:00-07:00',
          checkInTime: null,
          cancelled: false,
          numberOfParticipants: 3,
        },
        {
          room: {
            name: 'pandora edited',
          },
          eventTitle: 'ML&DP',
          endTime: '2019-06-13T06:30:00-07:00',
          checkedIn: false,
          startTime: '2019-06-13T04:15:00-07:00',
          checkInTime: null,
          cancelled: false,
          numberOfParticipants: 2,
        },
        {
          room: {
            name: 'Tortuga',
          },
          eventTitle: 'Resource Assignment',
          endTime: '2019-06-13T06:30:00-07:00',
          checkedIn: false,
          startTime: '2019-06-13T04:00:00-07:00',
          checkInTime: null,
          cancelled: false,
          numberOfParticipants: 3,
        },
        {
          room: {
            name: 'Tortuga',
          },
          eventTitle: null,
          endTime: '2019-06-13T03:00:00-07:00',
          checkedIn: false,
          startTime: '2019-06-13T02:00:00-07:00',
          checkInTime: null,
          cancelled: false,
          numberOfParticipants: 4,
        },
      ],
      pages: 10,
      hasPrevious: true,
      hasNext: true,
      queryTotal: 48,
    },
  };

  const errorData = 'GraphQL error: Events do not exist for the date range';

  QueryHelper.getAnalyticForDailyRoomsEvents.mockResolvedValue(errorData);
  QueryHelper.getUserDetails.mockResolvedValue(userData);

  beforeEach(() => {
    wrapper = mount(<AnalyticsActivityComponent {...props} />);
  });

  afterEach(() => {
    jest.restoreAllMocks();
  });

  it('renders loading state correctly and displays an overlay on top of a dummy data ', async () => {
    QueryHelper.getAnalyticForDailyRoomsEvents.mockResolvedValue(data);
    const component = wrapper.instance();
    await component.getAnalyticsForDailyRoomEvents(1, 1);
    jest.spyOn(component, 'setState').mockResolvedValue();
    component.setState({
      analyticsForDailyRoomEvents: data,
    });
    expect(wrapper.instance().setState).toBeCalled();
    expect(wrapper.find('Spinner').exists()).toBe(true);
    expect(wrapper.find('.centered').exists()).toBe(true);
    expect(wrapper.find('.meeting-title').exists()).toBe(true);
    expect(wrapper.find('.activity-info-row').exists()).toBe(true);
  });

  it('should re-render UI with a new set of data when date values are changed in props', () => {
    wrapper.setProps({
      dateValue: {
        startDate: moment('2019 28 Apr', 'YYYY-MM-DD'),
        endDate: moment('2019 28 Apr', 'YYYY-MM-DD'),
      },
    });
    expect(QueryHelper.getAnalyticForDailyRoomsEvents).toHaveBeenCalled();
  });

  it('should update the state of location with the logged in user location', async () => {
    await wrapper.instance().getLocation();
    expect(wrapper.state('location').length).toBe(7);
    expect(QueryHelper.getUserDetails).toHaveBeenCalled();
  });

  it('should render daily room events', async () => {
    const component = mount(<AnalyticsActivityComponent {...props} />);
    component.update();
    expect(component).toMatchSnapshot();
    expect(component.state().loading).toBe(true);
    expect(component.state().error).toBe(false);
  });
});
