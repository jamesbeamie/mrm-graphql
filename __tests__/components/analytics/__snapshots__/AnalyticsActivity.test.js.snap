// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AnalyticsActivity component should render daily room events 1`] = `
<AnalyticsActivity
  dateValue={
    Object {
      "endDate": null,
      "startDate": null,
    }
  }
  queryCompleted={
    [MockFunction] {
      "calls": Array [
        Array [
          "analyticsForDailyRoomEvents",
        ],
        Array [
          "analyticsForDailyRoomEvents",
        ],
        Array [
          "analyticsForDailyRoomEvents",
        ],
        Array [
          "analyticsForDailyRoomEvents",
        ],
        Array [
          "analyticsForDailyRoomEvents",
        ],
        Array [
          "analyticsForDailyRoomEvents",
        ],
      ],
    }
  }
>
  <center
    className="room__events"
  >
    <div
      className="meeting-title"
    >
      <div
        className="title-heading"
      >
        Meeting Title
      </div>
      <div
        className="room-heading"
      >
        Booked Room
      </div>
      <div
        className="status-heading"
      >
        Status
      </div>
      <div
        className="participant-heading"
      >
        Participants
      </div>
    </div>
    <Overlay
      id="average-meeting"
    >
      <div>
        <div
          className="overlay"
          id="average-meeting"
        />
        <div
          className="Pagination-spinner"
        >
          <Spinner
            size=""
          >
            <div
              className="centered"
            >
              <div
                className="spinner"
              />
            </div>
          </Spinner>
        </div>
      </div>
    </Overlay>
    <div>
      <div
        className="activity"
        key="0034cd35-0bea-425a-942c-3cc9db2e0317"
      >
        <div
          className="activity-day"
        >
          Thu May 30, 2019
        </div>
        <div
          className="activity-info-row"
          key="6c79a030-f726-4364-9a1d-9a23f5b51dc3"
        >
          <div
            className="title"
          >
            <div>
              Measure Standup
            </div>
          </div>
          <div
            className="room"
          >
            Ikogosi
          </div>
          <div
            className="status"
          >
            <div
              className="started"
            >
              Started 
              <bdi>
                04:15:00
                 
                EAT
              </bdi>
            </div>
            <div
              className="ended"
            >
              Ended 
              <bdi>
                04:45:00
                 
                EAT
              </bdi>
            </div>
          </div>
          <div
            className="participants"
          >
            <img
              alt=""
              src="https://firebasestorage.googleapis.com/v0/b/learning-map-app.appspot.com/o/MrmFrontImages%2FGroup.svg?alt=media&token=7f06b0bb-46bb-456a-ac35-9eeab8a16362"
            />
            <span>
              10
            </span>
          </div>
        </div>
      </div>
      <div
        className="activity"
        key="d60b0728-589b-430c-94b1-3a910e98b259"
      >
        <div
          className="activity-day"
        >
          Thu May 30, 2019
        </div>
        <div
          className="activity-info-row"
          key="16c751ed-a526-488c-b807-dccf7922dd41"
        >
          <div
            className="title"
          >
            <div>
              Olamide x David
            </div>
          </div>
          <div
            className="room"
          >
            Safari5
          </div>
          <div
            className="status"
          >
            <div
              className="started"
            >
              Started 
              <bdi>
                12:00:00
                 
                EAT
              </bdi>
            </div>
            <div
              className="ended"
            >
              Ended 
              <bdi>
                01:00:00
                 
                EAT
              </bdi>
            </div>
          </div>
          <div
            className="participants"
          >
            <img
              alt=""
              src="https://firebasestorage.googleapis.com/v0/b/learning-map-app.appspot.com/o/MrmFrontImages%2FGroup.svg?alt=media&token=7f06b0bb-46bb-456a-ac35-9eeab8a16362"
            />
            <span>
              5
            </span>
          </div>
        </div>
      </div>
      <div
        className="activity"
        key="f8e1e14c-b8ff-4930-b6db-12a45dc233d6"
      >
        <div
          className="activity-day"
        >
          Thu May 30, 2019
        </div>
        <div
          className="activity-info-row"
          key="b97cb5d3-0740-49a6-8133-8f5469b411d0"
        >
          <div
            className="title"
          >
            <div>
              Interview: Oluwagbemileke Oguntayo  - Senior Operations Manager, Nigeria
            </div>
          </div>
          <div
            className="room"
          >
            Kajuru
          </div>
          <div
            className="status"
          >
            <div
              className="started"
            >
              Started 
              <bdi>
                12:00:00
                 
                EAT
              </bdi>
            </div>
            <div
              className="ended"
            >
              Ended 
              <bdi>
                01:00:00
                 
                EAT
              </bdi>
            </div>
          </div>
          <div
            className="participants"
          >
            <img
              alt=""
              src="https://firebasestorage.googleapis.com/v0/b/learning-map-app.appspot.com/o/MrmFrontImages%2FGroup.svg?alt=media&token=7f06b0bb-46bb-456a-ac35-9eeab8a16362"
            />
            <span>
              4
            </span>
          </div>
        </div>
      </div>
      <div
        className="activity"
        key="8620f86d-7f83-4ab6-a8ba-0741cb89321a"
      >
        <div
          className="activity-day"
        >
          Thu May 30, 2019
        </div>
        <div
          className="activity-info-row"
          key="c689cfcd-ae28-45b3-acd4-a0458a60389d"
        >
          <div
            className="title"
          >
            <div>
              Agnes / Annabelle Catch Up
            </div>
          </div>
          <div
            className="room"
          >
            Safari-me
          </div>
          <div
            className="status"
          >
            <div
              className="started"
            >
              Started 
              <bdi>
                03:30:00
                 
                EAT
              </bdi>
            </div>
            <div
              className="ended"
            >
              Ended 
              <bdi>
                04:00:00
                 
                EAT
              </bdi>
            </div>
          </div>
          <div
            className="participants"
          >
            <img
              alt=""
              src="https://firebasestorage.googleapis.com/v0/b/learning-map-app.appspot.com/o/MrmFrontImages%2FGroup.svg?alt=media&token=7f06b0bb-46bb-456a-ac35-9eeab8a16362"
            />
            <span>
              3
            </span>
          </div>
        </div>
      </div>
      <div
        className="activity"
        key="7afa1ae4-c421-4548-a748-7e4d64c00ef7"
      >
        <div
          className="activity-day"
        >
          Thu May 30, 2019
        </div>
        <div
          className="activity-info-row"
          key="a7c892aa-f771-4a4d-afa5-82604285b012"
        >
          <div
            className="title"
          >
            <div>
              Apprenticeship Sync
            </div>
          </div>
          <div
            className="room"
          >
            Cotonous
          </div>
          <div
            className="status"
          >
            <div
              className="started"
            >
              Started 
              <bdi>
                02:30:00
                 
                EAT
              </bdi>
            </div>
            <div
              className="ended"
            >
              Ended 
              <bdi>
                03:30:00
                 
                EAT
              </bdi>
            </div>
          </div>
          <div
            className="participants"
          >
            <img
              alt=""
              src="https://firebasestorage.googleapis.com/v0/b/learning-map-app.appspot.com/o/MrmFrontImages%2FGroup.svg?alt=media&token=7f06b0bb-46bb-456a-ac35-9eeab8a16362"
            />
            <span>
              59
            </span>
          </div>
        </div>
      </div>
      <div
        className="activity"
        key="b253d06a-2691-44e1-a3ab-72320d0b22bb"
      >
        <div
          className="activity-day"
        >
          Thu May 30, 2019
        </div>
        <div
          className="activity-info-row"
          key="6315254d-1321-4375-acb0-b6c73efe78ea"
        >
          <div
            className="title"
          >
            <div>
              Interview: Oluwatoyin Williams-Edem - Senior Manager, HR Business Partner
            </div>
          </div>
          <div
            className="room"
          >
            Safari-me
          </div>
          <div
            className="status"
          >
            <div
              className="started"
            >
              Started 
              <bdi>
                11:00:00
                 
                EAT
              </bdi>
            </div>
            <div
              className="ended"
            >
              Ended 
              <bdi>
                12:00:00
                 
                EAT
              </bdi>
            </div>
          </div>
          <div
            className="participants"
          >
            <img
              alt=""
              src="https://firebasestorage.googleapis.com/v0/b/learning-map-app.appspot.com/o/MrmFrontImages%2FGroup.svg?alt=media&token=7f06b0bb-46bb-456a-ac35-9eeab8a16362"
            />
            <span>
              3
            </span>
          </div>
        </div>
      </div>
      <div
        className="activity"
        key="a9ad8f39-69ff-4dc6-a5c4-2708edf89d0c"
      >
        <div
          className="activity-day"
        >
          Thu May 30, 2019
        </div>
        <div
          className="activity-info-row"
          key="e2be4b04-696b-45fb-9eee-dd87b94ce473"
        >
          <div
            className="title"
          >
            <div>
              Apprenticeship Sync
            </div>
          </div>
          <div
            className="room"
          >
            Cotonous
          </div>
          <div
            className="status"
          >
            <div
              className="started"
            >
              Started 
              <bdi>
                02:30:00
                 
                EAT
              </bdi>
            </div>
            <div
              className="ended"
            >
              Ended 
              <bdi>
                03:30:00
                 
                EAT
              </bdi>
            </div>
          </div>
          <div
            className="participants"
          >
            <img
              alt=""
              src="https://firebasestorage.googleapis.com/v0/b/learning-map-app.appspot.com/o/MrmFrontImages%2FGroup.svg?alt=media&token=7f06b0bb-46bb-456a-ac35-9eeab8a16362"
            />
            <span>
              59
            </span>
          </div>
        </div>
      </div>
      <div
        className="activity"
        key="9b459825-84a3-4774-bd6a-f37fb86be18a"
      >
        <div
          className="activity-day"
        >
          Thu May 30, 2019
        </div>
        <div
          className="activity-info-row"
          key="934c2d7a-1c99-4de3-86e0-762db10fb05c"
        >
          <div
            className="title"
          >
            <div>
              Team meetings
            </div>
          </div>
          <div
            className="room"
          >
            Safari-me
          </div>
          <div
            className="status"
          >
            <div
              className="started"
            >
              Started 
              <bdi>
                01:00:00
                 
                EAT
              </bdi>
            </div>
            <div
              className="ended"
            >
              Ended 
              <bdi>
                02:30:00
                 
                EAT
              </bdi>
            </div>
          </div>
          <div
            className="participants"
          >
            <img
              alt=""
              src="https://firebasestorage.googleapis.com/v0/b/learning-map-app.appspot.com/o/MrmFrontImages%2FGroup.svg?alt=media&token=7f06b0bb-46bb-456a-ac35-9eeab8a16362"
            />
            <span>
              2
            </span>
          </div>
        </div>
      </div>
      <div
        className="activity"
        key="677e22e0-3e01-43a5-833c-6baf302084cc"
      >
        <div
          className="activity-day"
        >
          Thu May 30, 2019
        </div>
        <div
          className="activity-info-row"
          key="957c5ca0-9e21-4a8b-9e65-0c29ad5a205f"
        >
          <div
            className="title"
          >
            <div>
              Extended org sync
            </div>
          </div>
          <div
            className="room"
          >
            Cognito
          </div>
          <div
            className="status"
          >
            <div
              className="started"
            >
              Started 
              <bdi>
                01:00:00
                 
                EAT
              </bdi>
            </div>
            <div
              className="ended"
            >
              Ended 
              <bdi>
                01:40:00
                 
                EAT
              </bdi>
            </div>
          </div>
          <div
            className="participants"
          >
            <img
              alt=""
              src="https://firebasestorage.googleapis.com/v0/b/learning-map-app.appspot.com/o/MrmFrontImages%2FGroup.svg?alt=media&token=7f06b0bb-46bb-456a-ac35-9eeab8a16362"
            />
            <span>
              12
            </span>
          </div>
        </div>
      </div>
      <div
        className="activity"
        key="d5dd7805-ddeb-4d8a-b5ff-06d112a4050b"
      >
        <div
          className="activity-day"
        >
          Thu May 30, 2019
        </div>
        <div
          className="activity-info-row"
          key="88f120f7-2506-4be2-9b2e-857d9a80d295"
        >
          <div
            className="title"
          >
            <div>
              Ann &lt;&gt; Sam
            </div>
          </div>
          <div
            className="room"
          >
            Oculus
          </div>
          <div
            className="status"
          >
            <div
              className="started"
            >
              Started 
              <bdi>
                12:00:00
                 
                EAT
              </bdi>
            </div>
            <div
              className="ended"
            >
              Ended 
              <bdi>
                01:00:00
                 
                EAT
              </bdi>
            </div>
          </div>
          <div
            className="participants"
          >
            <img
              alt=""
              src="https://firebasestorage.googleapis.com/v0/b/learning-map-app.appspot.com/o/MrmFrontImages%2FGroup.svg?alt=media&token=7f06b0bb-46bb-456a-ac35-9eeab8a16362"
            />
            <span>
              5
            </span>
          </div>
        </div>
      </div>
      <div
        className="activity"
        key="e1f0bfed-baa5-434c-be5e-f2a80685ad80"
      >
        <div
          className="activity-day"
        >
          Thu May 30, 2019
        </div>
        <div
          className="activity-info-row"
          key="31beb619-b22b-4191-8ac2-aecd1964c5ce"
        >
          <div
            className="title"
          >
            <div>
              Interview: Ayodele Odufuye - Senior Manager, HR Business Partner
            </div>
          </div>
          <div
            className="room"
          >
            Safari-me
          </div>
          <div
            className="status"
          >
            <div
              className="started"
            >
              Started 
              <bdi>
                12:00:00
                 
                EAT
              </bdi>
            </div>
            <div
              className="ended"
            >
              Ended 
              <bdi>
                01:00:00
                 
                EAT
              </bdi>
            </div>
          </div>
          <div
            className="participants"
          >
            <img
              alt=""
              src="https://firebasestorage.googleapis.com/v0/b/learning-map-app.appspot.com/o/MrmFrontImages%2FGroup.svg?alt=media&token=7f06b0bb-46bb-456a-ac35-9eeab8a16362"
            />
            <span>
              3
            </span>
          </div>
        </div>
      </div>
      <div
        className="activity"
        key="04c27268-d7c0-4edb-8996-0ede8d8b3626"
      >
        <div
          className="activity-day"
        >
          Thu May 30, 2019
        </div>
        <div
          className="activity-info-row"
          key="bfa99448-0da0-4923-b0fc-168e60c77b0a"
        >
          <div
            className="title"
          >
            <div>
              Member Education
            </div>
          </div>
          <div
            className="room"
          >
            Oculus
          </div>
          <div
            className="status"
          >
            <div
              className="started"
            >
              Started 
              <bdi>
                02:00:00
                 
                EAT
              </bdi>
            </div>
            <div
              className="ended"
            >
              Ended 
              <bdi>
                03:30:00
                 
                EAT
              </bdi>
            </div>
          </div>
          <div
            className="participants"
          >
            <img
              alt=""
              src="https://firebasestorage.googleapis.com/v0/b/learning-map-app.appspot.com/o/MrmFrontImages%2FGroup.svg?alt=media&token=7f06b0bb-46bb-456a-ac35-9eeab8a16362"
            />
            <span>
              2
            </span>
          </div>
        </div>
      </div>
      <div
        className="activity"
        key="bc7d7bcd-3254-4d82-b33e-2db1128c7102"
      >
        <div
          className="activity-day"
        >
          Thu May 30, 2019
        </div>
        <div
          className="activity-info-row"
          key="75d3990e-9eec-4977-a0e3-46913c5c5226"
        >
          <div
            className="title"
          >
            <div>
              Engineer Self Check-in 
            </div>
          </div>
          <div
            className="room"
          >
            test-5
          </div>
          <div
            className="status"
          >
            <div
              className="started"
            >
              Started 
              <bdi>
                12:30:00
                 
                EAT
              </bdi>
            </div>
            <div
              className="ended"
            >
              Ended 
              <bdi>
                01:00:00
                 
                EAT
              </bdi>
            </div>
          </div>
          <div
            className="participants"
          >
            <img
              alt=""
              src="https://firebasestorage.googleapis.com/v0/b/learning-map-app.appspot.com/o/MrmFrontImages%2FGroup.svg?alt=media&token=7f06b0bb-46bb-456a-ac35-9eeab8a16362"
            />
            <span>
              6
            </span>
          </div>
        </div>
      </div>
      <div
        className="activity"
        key="36a74142-68a3-47e0-a3fc-b9f467511531"
      >
        <div
          className="activity-day"
        >
          Thu May 30, 2019
        </div>
        <div
          className="activity-info-row"
          key="d5446048-fdf9-479c-ae5a-1ffdc2382806"
        >
          <div
            className="title"
          >
            <div>
              Oladimeji + Tejumade _ Software Engineer (Values Alignment Call)
            </div>
          </div>
          <div
            className="room"
          >
            Ikogosi
          </div>
          <div
            className="status"
          >
            <div
              className="started"
            >
              Started 
              <bdi>
                03:30:00
                 
                EAT
              </bdi>
            </div>
            <div
              className="ended"
            >
              Ended 
              <bdi>
                04:30:00
                 
                EAT
              </bdi>
            </div>
          </div>
          <div
            className="participants"
          >
            <img
              alt=""
              src="https://firebasestorage.googleapis.com/v0/b/learning-map-app.appspot.com/o/MrmFrontImages%2FGroup.svg?alt=media&token=7f06b0bb-46bb-456a-ac35-9eeab8a16362"
            />
            <span>
              3
            </span>
          </div>
        </div>
      </div>
      <div
        className="activity"
        key="81361ab3-2d71-4e2b-a39e-6ccfbba1ab2f"
      >
        <div
          className="activity-day"
        >
          Thu May 30, 2019
        </div>
        <div
          className="activity-info-row"
          key="d3712725-c099-409b-acb3-d857cdfb5e7d"
        >
          <div
            className="title"
          >
            <div>
              Chux &lt;&gt; Godwin - Values Alignment Interview
            </div>
          </div>
          <div
            className="room"
          >
            Ikogosi
          </div>
          <div
            className="status"
          >
            <div
              className="started"
            >
              Started 
              <bdi>
                12:30:00
                 
                EAT
              </bdi>
            </div>
            <div
              className="ended"
            >
              Ended 
              <bdi>
                01:30:00
                 
                EAT
              </bdi>
            </div>
          </div>
          <div
            className="participants"
          >
            <img
              alt=""
              src="https://firebasestorage.googleapis.com/v0/b/learning-map-app.appspot.com/o/MrmFrontImages%2FGroup.svg?alt=media&token=7f06b0bb-46bb-456a-ac35-9eeab8a16362"
            />
            <span>
              3
            </span>
          </div>
        </div>
      </div>
      <div
        className="activity"
        key="aeab4a5e-8560-4f52-886e-c7e24c07cc90"
      >
        <div
          className="activity-day"
        >
          Thu May 30, 2019
        </div>
        <div
          className="activity-info-row"
          key="2b9aa4ae-ea26-4ad0-b0f9-41ff26087f77"
        >
          <div
            className="title"
          >
            <div>
              Brad &lt;&gt; Tope 1:1
            </div>
          </div>
          <div
            className="room"
          >
            Ikogosi
          </div>
          <div
            className="status"
          >
            <div
              className="started"
            >
              Started 
              <bdi>
                03:00:00
                 
                EAT
              </bdi>
            </div>
            <div
              className="ended"
            >
              Ended 
              <bdi>
                03:30:00
                 
                EAT
              </bdi>
            </div>
          </div>
          <div
            className="participants"
          >
            <img
              alt=""
              src="https://firebasestorage.googleapis.com/v0/b/learning-map-app.appspot.com/o/MrmFrontImages%2FGroup.svg?alt=media&token=7f06b0bb-46bb-456a-ac35-9eeab8a16362"
            />
            <span>
              3
            </span>
          </div>
        </div>
      </div>
      <div
        className="activity"
        key="6fc8f3f7-78a0-4790-b5ed-0a79aacde2d2"
      >
        <div
          className="activity-day"
        >
          Thu May 30, 2019
        </div>
        <div
          className="activity-info-row"
          key="448a4322-9877-46ac-8d1e-fbf8765abcb2"
        >
          <div
            className="title"
          >
            <div>
              Lara&lt;&gt;Nneka
            </div>
          </div>
          <div
            className="room"
          >
            Ikogosi
          </div>
          <div
            className="status"
          >
            <div
              className="started"
            >
              Started 
              <bdi>
                05:00:00
                 
                EAT
              </bdi>
            </div>
            <div
              className="ended"
            >
              Ended 
              <bdi>
                05:30:00
                 
                EAT
              </bdi>
            </div>
          </div>
          <div
            className="participants"
          >
            <img
              alt=""
              src="https://firebasestorage.googleapis.com/v0/b/learning-map-app.appspot.com/o/MrmFrontImages%2FGroup.svg?alt=media&token=7f06b0bb-46bb-456a-ac35-9eeab8a16362"
            />
            <span>
              4
            </span>
          </div>
        </div>
      </div>
      <div
        className="activity"
        key="d1a901f7-2b36-4426-a601-081336f65e52"
      >
        <div
          className="activity-day"
        >
          Thu May 30, 2019
        </div>
        <div
          className="activity-info-row"
          key="4622ff05-f8a9-45e6-8be4-940a409d6596"
        >
          <div
            className="title"
          >
            <div>
              Ghana General Manager Interview: Christiane Koenig
            </div>
          </div>
          <div
            className="room"
          >
            Empire State
          </div>
          <div
            className="status"
          >
            <div
              className="started"
            >
              Started 
              <bdi>
                11:00:00
                 
                EAT
              </bdi>
            </div>
            <div
              className="ended"
            >
              Ended 
              <bdi>
                11:50:00
                 
                EAT
              </bdi>
            </div>
          </div>
          <div
            className="participants"
          >
            <img
              alt=""
              src="https://firebasestorage.googleapis.com/v0/b/learning-map-app.appspot.com/o/MrmFrontImages%2FGroup.svg?alt=media&token=7f06b0bb-46bb-456a-ac35-9eeab8a16362"
            />
            <span>
              4
            </span>
          </div>
        </div>
      </div>
      <div
        className="activity"
        key="b7b67caa-400e-4eff-afcb-d80754171a80"
      >
        <div
          className="activity-day"
        >
          Thu May 30, 2019
        </div>
        <div
          className="activity-info-row"
          key="064bcc03-4643-44d0-b526-5b475fbf1a23"
        >
          <div
            className="title"
          >
            <div>
              Extended org sync
            </div>
          </div>
          <div
            className="room"
          >
            Cognito
          </div>
          <div
            className="status"
          >
            <div
              className="started"
            >
              Started 
              <bdi>
                01:00:00
                 
                EAT
              </bdi>
            </div>
            <div
              className="ended"
            >
              Ended 
              <bdi>
                01:40:00
                 
                EAT
              </bdi>
            </div>
          </div>
          <div
            className="participants"
          >
            <img
              alt=""
              src="https://firebasestorage.googleapis.com/v0/b/learning-map-app.appspot.com/o/MrmFrontImages%2FGroup.svg?alt=media&token=7f06b0bb-46bb-456a-ac35-9eeab8a16362"
            />
            <span>
              12
            </span>
          </div>
        </div>
      </div>
      <div
        className="activity"
        key="a40113cf-de91-451b-a8cc-61f07f51c661"
      >
        <div
          className="activity-day"
        >
          Thu May 30, 2019
        </div>
        <div
          className="activity-info-row"
          key="309220cc-6378-4d2f-a4be-715f636afdd2"
        >
          <div
            className="title"
          >
            <div>
              Member Education
            </div>
          </div>
          <div
            className="room"
          >
            Oculus
          </div>
          <div
            className="status"
          >
            <div
              className="started"
            >
              Started 
              <bdi>
                02:00:00
                 
                EAT
              </bdi>
            </div>
            <div
              className="ended"
            >
              Ended 
              <bdi>
                03:30:00
                 
                EAT
              </bdi>
            </div>
          </div>
          <div
            className="participants"
          >
            <img
              alt=""
              src="https://firebasestorage.googleapis.com/v0/b/learning-map-app.appspot.com/o/MrmFrontImages%2FGroup.svg?alt=media&token=7f06b0bb-46bb-456a-ac35-9eeab8a16362"
            />
            <span>
              2
            </span>
          </div>
        </div>
      </div>
      <div
        className="activity"
        key="167df4ba-98dc-4e86-b0c6-8ae2d59aa4fb"
      >
        <div
          className="activity-day"
        >
          Thu May 30, 2019
        </div>
        <div
          className="activity-info-row"
          key="66ee6ef9-2a96-4256-b6d4-30d9542a72ad"
        >
          <div
            className="title"
          >
            <div>
              Ann &lt;&gt; Sam
            </div>
          </div>
          <div
            className="room"
          >
            Oculus
          </div>
          <div
            className="status"
          >
            <div
              className="started"
            >
              Started 
              <bdi>
                12:00:00
                 
                EAT
              </bdi>
            </div>
            <div
              className="ended"
            >
              Ended 
              <bdi>
                01:00:00
                 
                EAT
              </bdi>
            </div>
          </div>
          <div
            className="participants"
          >
            <img
              alt=""
              src="https://firebasestorage.googleapis.com/v0/b/learning-map-app.appspot.com/o/MrmFrontImages%2FGroup.svg?alt=media&token=7f06b0bb-46bb-456a-ac35-9eeab8a16362"
            />
            <span>
              5
            </span>
          </div>
        </div>
      </div>
      <div
        className="activity"
        key="a67ac7e5-dfae-4be2-a860-e1d79e5dc84e"
      >
        <div
          className="activity-day"
        >
          Thu May 30, 2019
        </div>
        <div
          className="activity-info-row"
          key="024d30b4-8947-406f-be6c-9282b700ebde"
        >
          <div
            className="title"
          >
            <div>
              Onsite Interview: Douglas + Team | Recruiting Operations Associate
            </div>
          </div>
          <div
            className="room"
          >
            Algiers
          </div>
          <div
            className="status"
          >
            <div
              className="started"
            >
              Started 
              <bdi>
                12:00:00
                 
                EAT
              </bdi>
            </div>
            <div
              className="ended"
            >
              Ended 
              <bdi>
                01:00:00
                 
                EAT
              </bdi>
            </div>
          </div>
          <div
            className="participants"
          >
            <img
              alt=""
              src="https://firebasestorage.googleapis.com/v0/b/learning-map-app.appspot.com/o/MrmFrontImages%2FGroup.svg?alt=media&token=7f06b0bb-46bb-456a-ac35-9eeab8a16362"
            />
            <span>
              5
            </span>
          </div>
        </div>
      </div>
      <div
        className="activity"
        key="2ad72caf-f55c-4db5-9be7-4b78b164354c"
      >
        <div
          className="activity-day"
        >
          Thu May 30, 2019
        </div>
        <div
          className="activity-info-row"
          key="c11cd3ea-ce37-4e6e-b3b2-a231b87d3be9"
        >
          <div
            className="title"
          >
            <div>
              Values Call Alignment: Caroline&lt;&lt;&gt;&gt; Manasseh
            </div>
          </div>
          <div
            className="room"
          >
            Algiers
          </div>
          <div
            className="status"
          >
            <div
              className="started"
            >
              Started 
              <bdi>
                01:00:00
                 
                EAT
              </bdi>
            </div>
            <div
              className="ended"
            >
              Ended 
              <bdi>
                02:00:00
                 
                EAT
              </bdi>
            </div>
          </div>
          <div
            className="participants"
          >
            <img
              alt=""
              src="https://firebasestorage.googleapis.com/v0/b/learning-map-app.appspot.com/o/MrmFrontImages%2FGroup.svg?alt=media&token=7f06b0bb-46bb-456a-ac35-9eeab8a16362"
            />
            <span>
              3
            </span>
          </div>
        </div>
      </div>
      <div
        className="activity"
        key="ffa3a852-27c4-477c-971d-9dedb5e6d1c8"
      >
        <div
          className="activity-day"
        >
          Thu May 30, 2019
        </div>
        <div
          className="activity-info-row"
          key="5cb988f3-272a-4f0a-9281-6c5eff1748c2"
        >
          <div
            className="title"
          >
            <div>
              Dev/Partner Meetings :)
            </div>
          </div>
          <div
            className="room"
          >
            Test Add
          </div>
          <div
            className="status"
          >
            <div
              className="started"
            >
              Started 
              <bdi>
                11:50:00
                 
                EAT
              </bdi>
            </div>
            <div
              className="ended"
            >
              Ended 
              <bdi>
                04:25:00
                 
                EAT
              </bdi>
            </div>
          </div>
          <div
            className="participants"
          >
            <img
              alt=""
              src="https://firebasestorage.googleapis.com/v0/b/learning-map-app.appspot.com/o/MrmFrontImages%2FGroup.svg?alt=media&token=7f06b0bb-46bb-456a-ac35-9eeab8a16362"
            />
            <span>
              2
            </span>
          </div>
        </div>
      </div>
    </div>
  </center>
</AnalyticsActivity>
`;
